packages/app/src/components/Randomizer/Randomizer.test.js

```diff
@@ -32,59 +34,112 @@ locationService.getLeg.mockResolvedValue({
 })
 
 
-test('new restaurant button exists', async () => {
-  const { queryByTestId } = await actRender(<Randomizer />)
-  const buttonElement = queryByTestId('randomizer-randomizeButton')
+test('button for starting the raffle exists', async () => {
+  const { queryByTestId } = await actRender(
+    <Randomizer
+      maxNumberOfRolls={10}
+      maxTimeBetweenRolls={1}
+      minTimeBetweenRolls={0}
+      resultWaitTime={0}
+    />
```

Comment:
Defining these separately for each test is just plain stupid. I should have either defined the values in constants or declared a custom react component `<TestRandomizer />` which just renders the `Randomizer` with these values. Now if any of these props change radically in the future, we need to change every single test in this file.